### ISSUE TITLE
Project Overview: Display Last Upload Timestamp

### ISSUE THREAD

**Issue by msymons** [DependencyTrack/dependency-track#281]
### Issue Type:
  - [ ] defect report
  - [X] enhancement request
  
### Current Behavior:

For a sample project, the "Overview" screen in v3.4.0 displays a summary table with the following 7 fields:

```
Components                         129
Vulnerable Components              3
Vulnerabilities                    3
Findings Audited                   0
Findings Audited %                 0%
Suppressed                         0
Last Measurement                   8 Feb 2019 at 17:49:18

```

### Expected Behavior:
I would like to see the following additional fields displayed in the summary table:
* Last Upload.
* Last Upload Type (BOM/Scan)

Ie, the same info that is already displayed on main dashboard.

This would make it easy to visually compare the upload and measurement timestamps.  This is useful to spot when the upload timestamp is a long time *before* the measurement timestamp.

For me, I had a last upload "4 Feb 2019 at 16:42:32".  If I could have seen this info in the context of the project overview, I would have quickly realized that THIS is my one project where the BOM is not automatically uploaded and needs a manual upload!

### Environment:
  
  - Dependency-Track Version: 3.4.0

**Comment by leec94** [comment]
currently in v4.11.5, in Overview there is a field "Last BOM Import" and "Last Measurement" with dates, does this complete this issue?

**Comment by msymons** [comment]
This was implemented in v4.7 via DependencyTrack/frontend/issues/147

**Comment by msymons** [comment]
This was implemented in v4.7 via DependencyTrack/frontend/issues/147
